uiowa/uiowa
# [Grad] Degree updates

Resolves #7602.

# How to test

- `ddev blt ds --site=grad.uiowa.edu && ddev drush @grad.local uli /admin/content/fragments`
- Create new or edit existing fragment and add MPAff to its list of degrees offered. Check that none of the possible degrees offered have periods in their names. Save fragment.
- Visit https://grad.uiowa.ddev.site/programs/graduate-programs. Check that MPAff degree is listed for the program you added it to. You can also double-check no periods are appearing.
